web-server-doc/web-server/scribblings/http.scrbl

```diff
@@ -277,16 +279,25 @@ transmission that the server @bold{will not catch}.}
 
 @defmodule[web-server/http/cookie]{
  This module provides functions to create cookies and responses that set them.
-
- @defproc[(make-cookie [name cookie-name?] [value cookie-value?]
-                       [#:comment comment (or/c false/c string?) #f]
-                       [#:domain domain (or/c false/c valid-domain?) #f]
-                       [#:max-age max-age (or/c false/c exact-nonnegative-integer?) #f]
-                       [#:path path (or/c false/c string?) #f]
-                       [#:expires expires (or/c false/c string?) #f]
-                       [#:secure? secure? (or/c false/c boolean?) #f])
+ 
+ @defproc[(make-cookie [name cookie-name?]
+                       [value cookie-value?]
+                       [#:comment comment any/c #f]
+                       [#:domain domain (or/c domain-value? #f) #f]
+                       [#:max-age max-age (or/c (and/c integer? positive?) #f) #f]
+                       [#:path path (or/c path/extension-value? #f) #f]
+                       [#:expires expires (or/c date? string? #f) #f]
+                       [#:secure? secure? any/c #f]
+                       [#:http-only? http-only? any/c #f]
+                       [#:extension extension (or/c path/extension-value? #f) #f])
           cookie?]{
-  Constructs a cookie with the appropriate fields.
+   Constructs a cookie with the appropriate fields.
+
+   This is a wrapper around @racket[make-cookie] from @racketmodname[net/cookies/server]
+   for backwords compatability. The @racket[comment] argument is ignored.
+   If @racket[expires] is given as a string, it should match
+   @link["https://tools.ietf.org/html/rfc7231#section-7.1.1.2"]{RFC 7231, Section 7.1.1.2},
+   in which case it will be converted to a @racket[date?] value.
```

Comment:
`net/cookies/server` specifies `(or/c date? #f)`. The old implementation of `make-cookie` specifies `(or/c false/c string?)`, which is inherited from `cookie:add-expires` in `net/cookie`. So, `net/cookies/server` doesn't accept a string, the wrapper `make-cookie` will convert it if possible or throw an exception.